priceloop_api/priceloop/extras/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .exceptions import ColumnMixedTypes, ColumnsMismatch, ColumnTypesMismatch, TypeNotFound
+from .helpers import get_priceloop_client
```

Comment:
not sure if `get_priceloop_client` should be part of the __init__ here, for the reasons we talked about in tech discussion.